oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundable.java

```diff
@@ -0,0 +1,52 @@
+package org.heigit.ohsome.oshdb;
+
+public interface OSHDBBoundable {
```

Comment:
if we're touching this topic:

Why don't we go ahead and change the internal coordinate representation from the 64 bit longs to 32 bit integers: As long as `GEOM_PRECISION` is not bigger than 1E7, 32 bits are sufficient to store coordinates.

This would leave the question how we want to name the corresponding getters for the internal representation (`int getMinLonLong()` would not make much sense IMO). Maybe `int getMinLonInternal` could be an option?

Reply:
I just check, within the oshdb there is really only one place where we actually access the double return value variants.
`OSHDBGeometryBuilder::getGeometry`

So my suggestion here is, to remove the double return value methods completely and rename/change the long return value variants to 
```
int minLon();
int minLat();
int maxLon();
int maxLat();
 ```
 This would make it clear, that the OSHDBBoundable/OSHDBoundingBox use OSM-Integer coordinates. If the user needs the double coordinates back, she has to use converter methods like `OSHDBGeometryBuilder::getGeometry/getEnvelope`

Reply:
sounds good. :+1:

Reply:
I will address this issue, changing long coordinates to integer, in a following up PR which include also the OSMCoordinates and conversions from (int/long to double) from PR #320

Reply:
OK